pilot/pkg/security/authz/converter/converter.go

```diff
@@ -103,80 +95,157 @@ const (
 	istioConfigMapKey = "mesh"
 )
 
-func NewUpgrader(k8sClient *kubernetes.Clientset, v1PolicyFiles, serviceFiles []string, meshConfigFile, istioNamespace, meshConfigMapName string) *Upgrader {
-	upgrader := Upgrader{
-		K8sClient:                          k8sClient,
-		V1PolicyFiles:                      v1PolicyFiles,
-		ServiceFiles:                       serviceFiles,
-		MeshConfigFile:                     meshConfigFile,
-		IstioNamespace:                     istioNamespace,
-		MeshConfigMapName:                  meshConfigMapName,
-		NamespaceToServiceToWorkloadLabels: make(map[string]ServiceToWorkloadLabels),
-		AuthorizationPolicies:              []model.Config{},
-	}
-	return &upgrader
-}
-
-// ConvertV1alpha1ToV1beta1 converts RBAC v1alphal1 to v1beta1 for local policy files.
-func (ug *Upgrader) ConvertV1alpha1ToV1beta1() error {
-	configsFromFile, err := getRbacConfigsFromFiles(ug.V1PolicyFiles)
+func NewConverter(k8sClient *kubernetes.Clientset, v1PolicyFiles, serviceFiles []string,
```

Comment:
Now might be a good time to rename this to `New` to avoid the stutter. Usage would then be `converter.New` instead of `converter.NewConverter`.

Reply:
Good point, updated.